AmazonChimeSDK/AmazonChimeSDK/internal/video/VideoClientProtocol.swift

```diff
@@ -11,7 +11,7 @@ import AmazonChimeSDKMedia
 import Foundation
 
 @objc public protocol VideoClientProtocol {
-    var delegate: VideoClientDelegate! { get set }
+    weak var delegate: VideoClientDelegate! { get set }
```

Comment:
Also avoid force unwrapping the delegate:  `VideoClientDelegate?`

Reply:
Sure

Reply:
Discussed offline, this is an improvement on fixing memory leak, which will be handled separately. Updated the code to still use the old way and add a TODO.